components/_patterns/01-atoms/02-text/00-headings/heading-1/heading-1.twig

```diff
@@ -1,17 +1,19 @@
 {#
 /**
  * Available variables:
+ * - h1_base_class - the base class name
  * - heading_1_modifiers - array of modifiers to add to the base classname
  * - heading_1_blockname - blockname prepended to the base classname
  * - heading_1_url - (optional) the url the heading should poing to
  * - heading_1 - the content of the heading (typically text)
  */
 #}
-<h1 class="heading-1{% for modifier in heading_1_modifiers %} heading-1--{{ modifier }}{% endfor %}{% if heading_1_blockname %} {{ heading_1_blockname }}__heading-1{% endif %}">
+<h1 class="{{ h1_base_class|default('heading-1') }}{% for modifier in heading_1_modifiers %} {{ h1_base_class|default('heading-1') }}--{{ modifier }}{% endfor %}{% if heading_1_blockname %} {{ heading_1_blockname }}__{{ h1_base_class|default('heading-1') }}{% endif %}">
   {% if heading_1_url %}
     {% include "@atoms/01-links/link/link.twig"
       with {
-        "link_blockname": "heading-1",
+        "link_base_class": heading_1_link_base_class,
```

Comment:
The variables are already documented, but the documentation should be verified to make sure all variables are in the comments, and used properly.